airbyte-integrations/bases/base-normalization/integration_tests/normalization_test_output/postgres/test_simple_streams/first_output/airbyte_incremental/scd/test_normalization/dedup_exchange_rate_scd.sql

```diff
@@ -42,15 +42,15 @@ scd_data as (
             "date" desc,
             _airbyte_emitted_at desc
       ) as _airbyte_end_at,
-      case when lag("date") over (
+      case when row_number() over (
         partition by "id", currency, cast(nzd as 
     varchar
 )
         order by
             "date" is null asc,
             "date" desc,
             _airbyte_emitted_at desc
-      ) is null  then 1 else 0 end as _airbyte_active_row,
```

Comment:
when all (multiple) rows for the same primary key have a cursor (date here) equal to NULL, then this will flag multiple rows as active...